### DISCUSSION TITLE
3.2.0如何开启develop mode

### DISCUSSION THREAD

**Discussion by zhangzhao0827** [apache/dolphinscheduler#15293] [Q&A] [answered]
执行execute file时，提示 --jvm=-Xms1G -Xmx1G: command not found，想要查看相关文件的内容，但提示文件清理了，如何开启develop mode使execute file不被清理

**Comment by leachli** [comment]
你好，问题解决了吗？可否加个沟通方式？

**Comment by leachli** [comment]
我也遇到相同的问题

**Comment by leachli** [comment, marked as answer]
文件删除开启 开发者模式，配置参数在 worker-server    的conf   common.property文件中修改 development.state=true
